### PR TITLE
Update/boleto

### PR DESCRIPTION
**Pull request by allangrds**
## Description

Closes https://github.com/pagarme/mercurio/issues/322

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: cypress/integration/custom/pages/confirmation.spec.js [hunk 1]
```diff
@@ -61,7 +61,8 @@ describe('Custom', () => {
 
           cy.wait(400)
 
-          cy.get('.success__footer > button:last').click()
+          //--------
```
Comment (augusto-jm-amaral) [inline cypress/integration/custom/pages/confirmation.spec.js]: why this comment?

File: src/containers/Checkout/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (augusto-jm-amaral) [inline src/containers/Checkout/index.js]: add inline pls.

File: src/containers/Checkout/index.js [hunk 3]
```diff
@@ -89,6 +90,25 @@ const isTransactionSuccess = status =>
...
+    'props',
+    'pageInfo',
+    'payment',
+    'type'
+  ], thisOfClass)
+
+  if(choosedPaymentType === 'boleto') {
+    return {
+      title: 'Aguardando pagamento',
+      subtitle: ' Seu boleto será gerado em breve',
```
Comment (augusto-jm-amaral) [inline src/containers/Checkout/index.js]: Is this space before words necessary?

File: cypress/integration/custom/pages/confirmation.spec.js (hunk 1 above)
Comment (allangrds) [reply cypress/integration/custom/pages/confirmation.spec.js]: Opssss

**Comment by allangrds** [comment]
Feito @augusto-jm-amaral @thiagommedeiros

File: package.json [hunk 4]
```diff
@@ -10,14 +10,14 @@
...
   "bugs": "https://github.com/pagarme/artis/issues",
   "dependencies": {
-    "cep-promise": "2.0.7",
-    "classnames": "2.2.5",
+    "cep-promise": "3.0.3",
+    "classnames": "2.2.6",
     "colr": "1.2.2",
     "copy-to-clipboard": "3.0.8",
-    "dotenv": "5.0.1",
+    "dotenv": "6.1.0",
```
Comment (augusto-jm-amaral) [inline package.json]: Acho bizarro utilizarmos uma lib para ler um .dotenv haha

**Pull request merged**